src/lib/utils/preconditions/PreconditionContainerArray.ts

```diff
@@ -28,52 +28,52 @@ export const enum PreconditionRunMode {
 }
 
 /**
- * The condition for a [[PreconditionContainerArray]].
+ * The condition for a {@link PreconditionContainerArray}.
  */
 export enum PreconditionRunCondition {
 	/**
-	 * Defines a condition where all the entries must pass. This uses [[PreconditionConditionAnd]].
+	 * Defines a condition where all the entries must pass. This uses {@link PreconditionConditionAnd}.
 	 * @since 1.0.0
 	 */
 	And,
 
 	/**
-	 * Defines a condition where at least one entry must pass. This uses [[PreconditionConditionOr]].
+	 * Defines a condition where at least one entry must pass. This uses {@link PreconditionConditionOr}.
 	 * @since 1.0.0
 	 */
 	Or
 }
 
 /**
- * Defines the detailed options for the [[PreconditionContainerArray]], where both the [[PreconditionRunMode]] and the
+ * Defines the detailed options for the {@link PreconditionContainerArray}, where both the {@link PreconditionRunMode} and the
  * entries can be defined.
  * @since 1.0.0
  */
 export interface PreconditionArrayResolvableDetails {
 	/**
-	 * The data that will be used to resolve [[IPreconditionContainer]] dependent of this one.
+	 * The data that will be used to resolve {@link IPreconditionContainer} dependent of this one.
 	 * @since 1.0.0
 	 */
 	entries: readonly PreconditionEntryResolvable[];
 
 	/**
-	 * The mode the [[PreconditionContainerArray]] will run.
+	 * The mode the {@link PreconditionContainerArray} will run.
 	 * @since 1.0.0
 	 */
 	mode: PreconditionRunMode;
 }
 
 /**
- * Defines the data accepted by [[PreconditionContainerArray]]'s constructor.
+ * Defines the data accepted by {@link PreconditionContainerArray}'s constructor.
  * @since 1.0.0
  */
 export type PreconditionArrayResolvable = readonly PreconditionEntryResolvable[] | PreconditionArrayResolvableDetails;
 
 /**
  * Defines the data accepted for each entry of the array.
  * @since 1.0.0
- * @seealso [[PreconditionArrayResolvable]]
- * @seealso [[PreconditionArrayResolvableDetails.entries]]
+ * @seealso {@link PreconditionArrayResolvable}
+ * @seealso {@link PreconditionArrayResolvableDetails.entries}
```

Comment:
Are those displayed correctly in both the editor and TypeDocs?

Reply:
IntelliSense:
![image](https://user-images.githubusercontent.com/4019718/121768730-010f2200-cb60-11eb-9764-9a6adf18e294.png)

Clicking IntelliSense link:

https://user-images.githubusercontent.com/4019718/121768826-a32f0a00-cb60-11eb-9320-92e18e7090e4.mp4



---


TypeDoc:
![image](https://user-images.githubusercontent.com/4019718/121768788-5d724180-cb60-11eb-8fb5-e37ec5102c2a.png)

Clicking TypeDoc link:

https://user-images.githubusercontent.com/4019718/121768813-8692d200-cb60-11eb-9d3e-71f02981f49d.mp4

Reply:
![wizard harry](https://user-images.githubusercontent.com/24852502/121770355-218faa00-cb69-11eb-9890-09ebe617f9bb.gif)